webapp/webpack.common.js

```diff
@@ -7,7 +7,8 @@ let config = {
   entry: {
     admin: "./src/main/webapp/vue-apps/Admin/main.js",
     callButtons: "./src/main/webapp/vue-apps/CallButtons/main.js",
-    webConferencingNotificationExtension: "./src/main/webapp/vue-apps/Web-conferencing-notifications/main.js"
+    webConferencingNotificationExtension: "./src/main/webapp/vue-apps/Web-conferencing-notifications/main.js",
+    webConferencing: './src/main/webapp/vue-apps/VideoConference/main.js'
```

Comment:
can you rename your folder : 
./src/main/webapp/vue-apps/SpaceAdministrationVisioConference/
instead of
./src/main/webapp/vue-apps/VideoConference/